django/middleware/csrf.py

```diff
@@ -196,6 +226,24 @@ def _set_token(self, request, response):
             # Set the Vary header since content varies with the CSRF cookie.
             patch_vary_headers(response, ('Cookie',))
 
+    def _origin_verified(self, request):
```

Comment:
I don't mind, but I'm not 100% sure why this method has the `_` prefix where others don't. 🤔
(This may have been covered in the discussion.)

Reply:
I merely followed the convention of other helper methods like `_allow()`, `_reject()`, `_get_token()` and `_set_token()`.